core/input/util.js

```diff
@@ -14,6 +14,20 @@ function isEdge() {
     return navigator && !!(/edge/i).exec(navigator.userAgent);
 }
 
+// Get the height of a un-styled line of text
+export function getScrollLineHeight() {
```

Comment:
Urgh. Is this really necessary? Can't we have something hard coded? Or evaluate some CSS?

Reply:
Depends on how accurate we want to be. This is the only way to get the correct value.

Reply:
Do we need, or even want, accurate value for this though? VNC doesn't concern itself with pixels or lines, so line height isn't the important value here. What we really want is the physical distance the scroll wheel (or finger on a touch pad) moved. And from what I understood Firefox always sent the same line count for the same physical distance, no matter the line height?

Reply:
That is absolutely true, I just verified it. Let's just use the default value (19) then.